frontend/src/app/productivityMetrics/page.tsx

```diff
@@ -4,7 +4,7 @@ import React from 'react'
 export default function ProductivityMetricsPage() {
   return (
     <div className="main-container">
-      <GridWasteMetrics />
+      <GridWasteMetrics />      
```

Comment:
Nit: why the added whitespace?

Reply:
I think that might've been an accident on my end. Reason 1/2 for not programing that late at night. It's fixed now.